transport_nantes/dashboard/urls.py

```diff
@@ -1,8 +1,15 @@
 from django.urls import path
-from .views import DashboardIndex, SignatureView
+from dashboard import views
 
 app_name = 'dashboard'
 urlpatterns = [
-    path('', DashboardIndex.as_view(), name='index'),
-    path('signature/', SignatureView.as_view(), name='signature'),
+    path('', views.DashboardIndex.as_view(), name='index'),
+    path('signature/', views.SignatureView.as_view(), name='signature'),
+    path('email-campaigns/', views.EmailCampaignsDashboardView.as_view(),
```

Comment:
I'd call this something like `'sent'`, but that's a detail.  And then it would let you choose (for marketing email) whether you want emails or press releases, and for transactional the various sorts of transactional emails.  A tab line at the top of the page is probably the right interface to do that, but I'm pretty open.

For transactional emails, I can't think of a pertinent grouping off hand.  We'd just like to see recent transactional mails sent.